app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class='authform'>
+  <h3>Create User</h3>
+    <%= form_for @user do |f| %>
```

Comment:
v nice! If you want to take a stab at a once over, you might consider using the rails bootstrap forms gem we use -- will let you simplify down to something like this:

```
bootstrap_form_for @user do |f|
  f.email_field :email, autofocus: true
  f.text_field :name
  ...
end
```